### ISSUE TITLE
FTM2 crash

### ISSUE THREAD

**Issue by ghost** [makeasnek/FindTheMag2#24]
FTM2 crashes on my Ubuntu PC with the following message.

```
Traceback (most recent call last):
  File "/home/daniel/FindTheMag2/main.py", line 2666, in <module>
    boinc_loop(False,rpc_client)
  File "/home/daniel/FindTheMag2/main.py", line 2279, in boinc_loop
    custom_sleep(30,rpc_client,dev_loop=dev_loop)  # There's no reason to loop through all projects more than once every 30 minutes
  File "/home/daniel/FindTheMag2/main.py", line 1744, in custom_sleep
    if loop.run_until_complete(is_boinc_crunching(boinc_rpc_client)):
  File "/usr/lib/python3.10/asyncio/base_events.py", line 646, in run_until_complete
    return future.result()
  File "/home/daniel/FindTheMag2/main.py", line 262, in is_boinc_crunching
    reply = await run_rpc_command(rpc_client, 'get_cc_status')
  File "/home/daniel/FindTheMag2/main.py", line 743, in run_rpc_command
    response = await rpc_client._request(req)
  File "/home/daniel/FindTheMag2/libs/pyboinc/rpc_client.py", line 78, in _request
    return await self._raw_client.request(req)
  File "/home/daniel/FindTheMag2/libs/pyboinc/_raw_client.py", line 82, in request
    return await self.receive()
  File "/home/daniel/FindTheMag2/libs/pyboinc/_raw_client.py", line 63, in receive
    buff = [await self._read()]
  File "/home/daniel/FindTheMag2/libs/pyboinc/_raw_client.py", line 57, in _read
    return await self._reader.readuntil(separator=END_OF_MESSAGE)
  File "/usr/lib/python3.10/asyncio/streams.py", line 614, in readuntil
    raise exceptions.IncompleteReadError(chunk, None)
asyncio.exceptions.IncompleteReadError: 0 bytes read on a total of undefined expected bytes
```

**Comment by HeyMerlin** [comment]
Do you have any more detail on your setup and reproducing the error...
Does the crash happen consistently?
Was this the first run or has FTM been running successfully?
Are the BOINC client and Gridcoin wallet installed on this machine?
Do you have a default `config.py` file or have you customized it?

**Comment by Frigyes06** [comment]
@sweede-se Did you fix this issue?

**Comment by sweede-se** [comment]
Default config, Boinc and wallet on the machine.

**Comment by makeasnek** [comment]
Fixed in latest release v 3.0
